src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -24,13 +24,14 @@
 /**
  * A connection used to invoke hub methods on a SignalR Server.
  */
-public class HubConnection {
+public class HubConnection implements AutoCloseable {
```

Comment:
@anurse should we go through API review for this, or do it among the SignalR folks?

Reply:
It probably should go through API review but it’s pretty minor. Could be done on email. Is this basically Java’s “IDisposable”? If so, I don’t see any concern with adding it.

Reply:
> Is this basically Java’s “IDisposable”?

Basically
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html